src/MediaWiki/Search/ProfileForm/Forms/OpenForm.php

```
@@ -33,28 +30,22 @@ public function __construct( private readonly WebRequest $request ) {
-		$this->isActiveForm = (bool)$isActiveForm;
+	public function isActiveForm( bool $isActiveForm ): void {
+		$this->isActiveForm = $isActiveForm;
 	}
 
 	/**
 	 * @since 3.0
-	 *
-	 * @param array $definition
 	 */
-	public function makeFields( $definition = [] ): string {
+	public function makeFields( array $definition = [] ): string {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$definition'. (undefined)

(UnusedFormalParameter)